src/doc/trpl/installing-rust.md

```diff
@@ -1,24 +1,24 @@
 % Installing Rust
 
 The first step to using Rust is to install it! There are a number of ways to
-install Rust, but the easiest is to use the `rustup` script. If you're on Linux
-or a Mac, all you need to do is this (note that you don't need to type in the
-`$`s, they just indicate the start of each command):
+install Rust, but first we ought to go over the officially supported architecture's.
```

Comment:
Hello Steve! I put it this way because I think the first question one might ask when looking through product documentation is "Will Rust work for me?" I think setting the scope of things currently supported is a good place to start the conversation, if I was trying to sell Rust I would start by saying how it already works with what folks have and why it not only augments but streamlines their current production environment.  It's only in this order for that sort of anecdotal reason.